static/app/views/issueList/customViewsHeader.tsx

```diff
@@ -361,6 +369,7 @@ function CustomViewsIssueListHeaderTabsContent({
     queryParamsWithPageFilters,
     draggableTabs,
     organization,
+    tempTab,
```

Comment:
Is `tempTab` used in this useEffect? doesn't seem like it needs to be a dependency

Reply:
oops good catch - this was left in from a previous change, removing it now